rbx_xml/src/deserializer.rs

```diff
@@ -315,6 +324,72 @@ fn deserialize_metadata<R: Read>(reader: &mut XmlEventReader<R>, state: &mut Par
     Ok(())
 }
 
+fn deserialize_shared_string_dict<R: Read>(
+    reader: &mut XmlEventReader<R>,
+    state: &mut ParseState,
+) -> Result<(), DecodeError> {
+    reader.expect_start_with_name("SharedStrings")?;
+
+    loop {
+        match reader.expect_peek()? {
+            XmlReadEvent::StartElement { name, .. } => {
+                if name.local_name == "SharedString" {
+                    deserialize_shared_string(reader, state)?;
+                } else {
+                    let event = reader.expect_next().unwrap();
+                    return Err(reader.error(DecodeErrorKind::UnexpectedXmlEvent(event)));
+                }
+            }
+            XmlReadEvent::EndElement { name } => {
+                if name.local_name == "SharedStrings" {
```

Comment:
Typo?

Reply:
Actually not a typo, that's the name of the `SharedString` dictionary in the file, `SharedStrings`!